certified-connectors/DocuSignDemo/apiDefinition.swagger.json

```diff
@@ -1356,6 +1356,84 @@
         "x-ms-visibility": "advanced"
       }
     },
+    "/accounts/{accountId}/envelopes/{envelopeId}/recipientsV2": {
```

Comment:
Lets not version it as V2 because, it can lead to confusion that we might deprecate the other recipients endpoint. 
Instead we can call it recipientFields